From a480b918fdfa7e1390bb8a0c3e517593b307f3da Mon Sep 17 00:00:00 2001
From: Steven Casagrande <stevencasagrande@gmail.com>
Date: Tue, 5 Mar 2024 21:19:33 -0500
Subject: [PATCH] Remove upward folder traversal

---
 src/rd.h          | 2 +-
 src/tinycthread.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/rd.h b/src/rd.h
index fd6c307f..0f4cd99b 100644
--- a/src/rd.h
+++ b/src/rd.h
@@ -61,7 +61,7 @@
 #include "win32_config.h"
 #else
 /* POSIX / UNIX based systems */
-#include "../config.h" /* mklove output */
+#include "config.h" /* mklove output */
 #endif
 
 #ifdef _WIN32
diff --git a/src/tinycthread.h b/src/tinycthread.h
index 6bc39fe0..0590b098 100644
--- a/src/tinycthread.h
+++ b/src/tinycthread.h
@@ -29,7 +29,7 @@ freely, subject to the following restrictions:
 #ifdef _WIN32
 #include "win32_config.h"
 #else
-#include "../config.h"
+#include "config.h"
 #endif
 
 #if WITH_C11THREADS
-- 
2.34.1

